lesson-6/style/style.css

```diff
@@ -0,0 +1,132 @@
+* {
+  margin: 0; 
+  padding: 0;
+  box-sizing: border-box;
+  }
+  .rectangle52 {
+   width: 1440px;
+   height: 900px;
+   min-height: 100%;
+   background-image: url(style/img/castelmezzano-1979546\ 1.svg);
+   background-color: lightcoral;
+   background-repeat: no-repeat;
+   background-position: 0 0;
+  }
+  header {
+   width: 1232px;
+   height: 46px;
+   margin: 51px 104px 196px 104px;
```

Comment:
51px is not needed
![image](https://user-images.githubusercontent.com/20189313/127404321-11e8bf61-f033-4758-abcd-7066b7f0a8f6.png)